### ISSUE TITLE
Add logging to backend

### ISSUE THREAD

**Issue by maximekuhn** [Meliodas-1O/modernMoodle#28]
We should add logs to our backend.
For instance we could have logs to know which endpoints were called.
We might have something like:

`request (from the client) -> logger -> controller`

**Comment by Meliodas-1O** [comment]
Yeah, it might be helpful to identify future bugs !

**Comment by maximekuhn** [comment]
yes and it will also allow us to see if any route is much more used than the others
